phpunit/block-supports/layout-test.php

```diff
@@ -380,4 +380,88 @@ public function data_gutenberg_get_layout_style() {
 			),
 		);
 	}
+
+	/**
+	 * Check that gutenberg_render_layout_support_flag() renders the correct classnames on the wrapper.
+	 *
+	 * @dataProvider data_layout_support_flag_renders_classnames_on_wrapper
+	 *
+	 * @covers ::gutenberg_render_layout_support_flag
+	 *
+	 * @param array  $args            Dataset to test.
+	 * @param string $expected_output The expected output.
+	 */
+	public function test_layout_support_flag_renders_classnames_on_wrapper( $args, $expected_output ) {
+		$actual_output = gutenberg_render_layout_support_flag( $args['block_content'], $args['block'] );
+		$this->assertEquals( $expected_output, $actual_output );
+	}
+
+	/**
+	 * Data provider for test_layout_support_flag_renders_classnames_on_wrapper.
+	 *
+	 * @return array
+	 */
+	public function data_layout_support_flag_renders_classnames_on_wrapper() {
+		return array(
+			'single wrapper block layout with flow type'   => array(
+				'args'            => array(
```

Comment:
This looks good to me, but I was wondering if there's a code style preference between placing the params in a nested `args` array or moving them one level up to be at the root of the array (and therefore become their own params on `test_layout_support_flag_renders_classnames_on_wrapper`)? So far I haven't been nesting them further in data providers, but it does look like a good way to keep the params contained 👍

In short: this was just a comment, not anything that needs changing IMO 🙂

Reply:
I copied this from the `get_layout_style` tests as it felt like a nice way to structure it! Not sure how widespread one or the other approach is.

Reply:
Ah yes, of course! Let's leave it as is, looks nice to me, too, I'll probably borrow it next time I'm writing a data provider 🙂